ml-agents-envs/mlagents/envs/environment.py

```diff
@@ -288,55 +280,18 @@ def executable_launcher(self, file_name, docker_training, no_graphics, args):
                 )
 
     def __str__(self):
-        reset_params_str = (
```

Comment:
Bad merge. Don't try to print the brains here because it's always empty at startup.

Reply:
Thinking about removing the __str__ altogether in another PR. Thoughts ?

Reply:
Yeah, maybe, although the logs might be informative.